src/js/actions/tools.js

```diff
@@ -755,7 +755,7 @@ define(function (require, exports) {
             }
         }
 
-        headlights.logEvent("tools", "mask-mode-" + currentLayer.kind);
+        headlights.logEvent("tools", "mask-mode", (currentLayer.kind).toString());
```

Comment:
Err... I'm concerned that `Layer.prototype.kind` might be null, in which case this will throw an error. (Note [here](https://github.com/adobe-photoshop/spaces-design/blob/084d00c6ebe4be31d2b51bc94cf46eac812f5398/src/js/actions/layers.js#L92) that the `layerKind` property on the layer descriptor is marked "optional", which almost certainly means that there is some sort of layer which does not have a kind. E.g., background layers? Who knows.) Alternatives: `String(currentLayer.kind)` or `"" + currentLayer.kind`.